src/largest-number.test.js

```diff
@@ -0,0 +1,30 @@
+const largestNumber = require('./largest-number');
```

Comment:
you are missing one test case.
pls add one test case that given number1 as a largest value.
example: 100, 0, 1
example: 50, 0, 0